src/Dockerfile

```diff
@@ -46,13 +46,14 @@ RUN cargo +1.49.0 install wasm-pack
 RUN cargo +1.49.0 install cargo-tree
 
 # Install latest node
-RUN sudo curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
-RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
-RUN sudo apt-get update && sudo apt-get install -y nodejs yarn
+RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
+RUN apt-get -y install nodejs
+RUN npm install -g yarn
 
+RUN node -v
 RUN yarn --version
-RUN sudo yarn global add n
-RUN sudo n latest -q
+RUN yarn global add n
+RUN n latest -q
```

Comment:
I think we should go with stable here, not latest

Reply:
mmm, i just leave everything the same way it was. Just change the base docker image. Do you want me to change it anyway?

Reply:
Yes, I think it is a good opportunity to review these things.

Reply:
Done!